sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -69,10 +72,14 @@ def _get_instance_types(region: str) -> pd.DataFrame:
 
 
 @ray.remote
-def _get_availability_zones(region: str) -> pd.DataFrame:
-    client = aws.client('ec2', region_name=region)
-    zones = []
-    response = client.describe_availability_zones()
+def _get_availability_zones(region: str) -> Optional[pd.DataFrame]:
+    try:
+        client = aws.client('ec2', region_name=region)
+        zones = []
+        response = client.describe_availability_zones()
+    except aws.client_exception():
+        # The user's AWS account may not have access to this region.
```

Comment:
nit: Maybe add a comment of an example exception message?

Reply:
Good point! Added. Thanks!